test/puppetlabs/pcp/message_test.clj

```diff
@@ -10,31 +10,16 @@
     (is (= "☃" (-> "☃" string->bytes bytes->string)))))
 
 (deftest make-message-test
-  (testing "the created message is a Message"
-    (is (s/validate Message (make-message))))
-  (testing "it makes a message"
-    (is (= {:sender ""
-            :targets []
-            :expires "1970-01-01T00:00:00.000Z"
-            :message_type ""}
-           (dissoc (make-message) :_chunks :id))))
   (testing "it makes a message with parameters"
-    (let [message (make-message :sender "pcp://client01.example.com/test"
-                                :targets ["pcp:///server"])]
+    (let [message (make-versioned-message
+                    "v2.0"
+                    {:sender "pcp://client01.example.com/test"
+                     :target ["pcp:///server"]})]
```

Comment:
Oh, it expects an array. But the equality check below tries to match against an array, which is no longer true.